python/lsst/ip/diffim/makeRatingVector.py

```diff
@@ -48,32 +47,32 @@ def makeRatingVector(kernelCellSet, spatialKernel, spatialBg):
                 diffIm = cand.getDifferenceImage(kernel, background)
                 imstats.apply(diffIm)
 
-                #candMean   = imstats.getMean()
-                #candRms    = imstats.getRms()
-                #candRating = sdqa.SdqaRating("lsst.ip.diffim.residuals_%d_%d" % (xCand, yCand),
-                #                             candMean, candRms, scope)
-                #sdqaVector.append(candRating)
+                # candMean   = imstats.getMean()
+                # candRms    = imstats.getRms()
+                # candRating = sdqa.SdqaRating("lsst.ip.diffim.residuals_%d_%d" % (xCand, yCand),
+                #                              candMean, candRms, scope)
+                # sdqaVector.append(candRating)
             elif cand.getStatus() == afwMath.SpatialCellCandidate.BAD:
                 nBad += 1
 
-    #nGoodRating = sdqa.SdqaRating("lsst.ip.diffim.nCandGood", nGood, 0, scope)
-    #sdqaVector.append(nGoodRating)
-    #nBadRating = sdqa.SdqaRating("lsst.ip.diffim.nCandBad", nBad, 0, scope)
-    #sdqaVector.append(nBadRating)
+    # nGoodRating = sdqa.SdqaRating("lsst.ip.diffim.nCandGood", nGood, 0, scope)
+    # sdqaVector.append(nGoodRating)
+    # nBadRating = sdqa.SdqaRating("lsst.ip.diffim.nCandBad", nBad, 0, scope)
+    # sdqaVector.append(nBadRating)
 
     nKernelTerms = spatialKernel.getNSpatialParameters()
-    if nKernelTerms == 0: # order 0
+    if nKernelTerms == 0:  # order 0
         nKernelTerms = 1
-    #nBgTerms     = len(spatialBg.getParameters())
-    #nKernRating  = sdqa.SdqaRating("lsst.ip.diffim.nTermsSpatialKernel", nKernelTerms, 0, scope)
-    #nBgRating    = sdqa.SdqaRating("lsst.ip.diffim.nTermsSpatialBg", nBgTerms, 0, scope)
-    #sdqaVector.append(nKernRating)
-    #sdqaVector.append(nBgRating)
+    # nBgTerms     = len(spatialBg.getParameters())
+    # nKernRating  = sdqa.SdqaRating("lsst.ip.diffim.nTermsSpatialKernel", nKernelTerms, 0, scope)
+    # nBgRating    = sdqa.SdqaRating("lsst.ip.diffim.nTermsSpatialBg", nBgTerms, 0, scope)
+    # sdqaVector.append(nKernRating)
+    # sdqaVector.append(nBgRating)
 
-    #for i in range(sdqaVector.size()):
-    #    pexLog.Trace("lsst.ip.diffim.makeSdqaRatingVector", 5,
-    #                 "Sdqa Rating %s : %.2f %.2f" % (sdqaVector[i].getName(),
-    #                                                 sdqaVector[i].getValue(),
-    #                                                 sdqaVector[i].getErr()))
+    # for i in range(sdqaVector.size()):
+    #     pexLog.Trace("lsst.ip.diffim.makeSdqaRatingVector", 5,
+    #                  "Sdqa Rating %s : %.2f %.2f" % (sdqaVector[i].getName(),
+    #                                                  sdqaVector[i].getValue(),
+    #                                                  sdqaVector[i].getErr()))
     #
-    #return sdqaVector
+    # return sdqaVector
```

Comment:
You are likely right. I'll see how old the code is and delete it if it's old, else ask around.